docs/tutorials/index.md

```diff
@@ -0,0 +1,29 @@
+# Tutorials
+
+The diagram below shows several G-ADOPT example codes and how they
+relate to each other.  These notebooks demonstrate how to use G-ADOPT
+to set up (forward and inverse) geodynamical simulations.
+
+```mermaid
+--8<-- "docs/tutorials/.diagram.mermaid"
+```
+
+We start with the most basic problem - isoviscous, incompressible
```

Comment:
In this paragraph possible worth adding in brackets the case names listed in the diagram:

(Base)
Compressible (ALA - TALA) etc...